mozilla_schema_generator/__main__.py

```diff
@@ -56,13 +56,15 @@
 )
 def generate_main_ping(config, out_dir, split, pretty):
     schema_generator = MainPing()
+    if out_dir:
```

Comment:
It might be useful to mention somewhere that the `out_dir` corresponds to the schema namespace.